app/utils/planUtils.ts

```diff
@@ -13,11 +13,21 @@ export function isEnterprisePlan(planType: PayPlanType) {
   return planType === PayPlanType.Enterprise
 }
 
+export function isUnlimitedPlan(planType: PayPlanType) {
+  return planType === PayPlanType.PlanUnlimited
```

Comment:
@commoddity I haven't followed on all the edge cases, but do we want/need to do an `|| planType === PayPlanType.Enterprise` here?